test/destest.c

```diff
@@ -287,7 +287,7 @@ static char *pt(const unsigned char *p, char buf[DATA_BUF_SIZE])
 {
     char *ret;
     int i;
-    static char *f = "0123456789ABCDEF";
+    static const char *f = "0123456789ABCDEF";
```

Comment:
Wouldn't `static const char f[] = "0123456789ABCDEF";` be better?  It avoids a pointer.